orders/views.py

```diff
@@ -1,37 +1,49 @@
 import json
 
-from django.http     import JsonResponse
-from django.views    import View
-from django.db       import transaction
+from django.http      import JsonResponse
+from django.views     import View
+from django.db        import transaction
+from django.db.models import Q, F
 
-from products.models import Item
-from orders.models   import Order, OrderItem, Location
-from users.models    import Member
-from users.utils     import login
+from products.models  import Item
+from orders.models    import Order, OrderItem, Location, Cart
+from users.models     import Member
+from users.utils      import login
 
 class OrderView(View):
     @login
     @transaction.atomic
-    def post(self, request, item_id):
-        data = json.loads(request.body)
-        
+    def post(self, request):
+        item_id = request.GET.get("item_id", None)
+        data    = json.loads(request.body)      
+                
         if Order.objects.filter(member_id = request.user.id, status_id = 1).exists():
```

Comment:
`status_id = 1`로 하게되면, 1번이 의미하는게 무엇인지는 작성한 사람만 알 수 있습니다.
그래서 이런경우에는 Enum으로 status를 관리해야줘야 합니다. 아래 예제를 참고해서 만들어보세요.

예를 들면 아래와 같이 코드를 작성하셔야 합니다.
```python
class OrderStatus(models.Model):
    class Status(models.IntegerChoices):
        CANCLED   = 1
        COMPLETED = 2
```
!! 여기서 주의해야될 점은 `CANCLED   = 1`은 OrderStatus 테이블에 저장된 값과 동일하게 관리해야된다는 점입니다.
status table
```
id  |  name
1   | CANCLED
2   | COMPLETED
```

```suggestion
        if Order.objects.filter(member_id = request.user.id, status_id = OrderStatus.CANCLED.value).exists():
```

Reply:
수정내용 모두 반영했습니다~~!